docs/docs/modeling.md

```diff
@@ -106,6 +106,44 @@ classDiagram
 
 ## ShareHouse - シェアハウス
 
+The Landlord can register a ShareHouse which they own.
+
+### ShareHouse ID (Identifier)
+
+The ShareHouse ID is the UUID.
+
+### ShareHouse Name
+
+The ShareHouse has a default name when created without specifying one.
```

Comment:
I think we talked about `default name` and decided ask the user to name the sharehouse.